### DISCUSSION TITLE
OpenUpgrade don't run base modules migration scripts

### DISCUSSION THREAD

**Discussion by tonyctalope** [OCA/OpenUpgrade#3831] [Q&A] [answered]
As described in this issue (https://github.com/OCA/OpenUpgrade/issues/3830), it is likely that there is a problem with my deployment. If anyone can help me with that, I would be happy to hear from you.

**Last response** : 
Thanks for reporting. As the openupgrade 13.0 project works correctly in the CI and for other people, I suspect a problem in your deployment. If there is no log of the migrations, one of the basic origin is that the code executed is the odoo code and not the openupgrade code.
if the base is executed and not the other modules, I think there is a trouble in your addons path.
Did you tried to changed it ?

**My response** : 
I did not try to change it as I am not sure what I can change it to.
Here is my directory structure:
```
OpenUpgrade
|___addons
|    |___ account
|    |___ account_asset_management
|    |___ ... addons ...
|___ odoo
     |___ addons
          |___ base
```

My addons_path is /etc/odoo/OpenUpgrade/addons.
There may be a problem with the addons_path, but I am confused as to why the migration script for account_asset_management is running without any issues.

**Comment by pedrobaeza** [comment, marked as answer]
`base` module doesn't enter in the addons_path, so you need to change your Docker file to use the OpenUpgrade one, or use other deployment method like Doodba.

**Comment by echarp** [comment]
I encountered the same issue, and discovered that in this version of OpenUpgrade, the 13.0 branch, any previously installed addons, for example in `/usr/lib/python3/dist-packages/odoo/addons`, override the migrations.

Temporarily moving away this directory did the trick for me...

**Comment by adonisadipos** [comment]
I do in my Docker file this:

```
# move the original folder with addons away
# https://github.com/OCA/OpenUpgrade/discussions/3831
RUN mv /usr/lib/python3/dist-packages/odoo/addons /usr/lib/python3/dist-packages/odoo/addons.bak
```
And that was the solution.
